### ISSUE TITLE
DEV: Un-responsive reStructuredText preview via Gitpod

### ISSUE THREAD

**Issue by bhavukkalra** [numpy/numpy#21176]
### Issue with current documentation:

Issue in brief - When a user creates a new Gitpod session for building docs. The preview provided for `.rst` files by reStructuredText VS Code extension is not visible.

Doccumentation reference Gitpod for Documentation builds - [Development Gitpod](https://numpy.org/devdocs/dev/development_gitpod.html)

Issue - Opened a sample `.rst` file for demonstration purpose(but the issue is with each file)

**Blank Preview**  - 

![issue_describe](https://user-images.githubusercontent.com/41142068/157753383-4ae9f0a8-2dba-4213-b20c-f00d0b7042ce.png)

On-Further Analysis, found out that the issue is getting caused by default python environment for VS Code set to 
`Python3.8.10` not the required one `numpy-dev`

![recommended](https://user-images.githubusercontent.com/41142068/157756839-11781429-d321-4cd5-9331-2bbf1167e571.PNG)

When the VS Code environment is changed to `numpy-dev` the preview works perfectly. 

Problem - 
The Documentation does address that `numpy-dev and should be displayed in the status bar` but is not highlighted as much and could be missed by a developer.

The Line which needs highlight `displayed in the status bar`  (As it is not set by default and requires manual intervention)

![dont_highlight](https://user-images.githubusercontent.com/41142068/157758841-9ce45879-ceb1-4a07-b586-f08995442ce4.PNG)

**Proposed Solution 1** - Highlight this line(_displayed in the status bar_) by writing it in **BOLD** or using RED as font color to provide more attention to the reader.

**Proposed solution 2** - Add the above discussion of `Not able to render preview of Documentation files` in the FAQ section of [FAQ Section Developing via Gitpod](https://numpy.org/devdocs/dev/development_gitpod.html#faq-s-and-troubleshooting). Highlighting above points for Solution.

**Comment by rossbar** [comment]
IMO the preferred solution would be to set the `numpy-dev` env as the default. From the docs it seems like this is what was intended, but the gitpod instance must have gotten out of sync somehow. @melissawm  any idea how to set/configure the default env in gitpod?

**Comment by bhavukkalra** [comment]
Found a solution to this. Upon looking into it further I came to know that this is getting caused by the deprecated `vscode` python extension setting `python.pythonPath`. 
More Details Here - [Python Extension AB-Experiments](https://github.com/microsoft/vscode-python/wiki/AB-Experiments).

**FIX** - replace the `python.pythonPath` setting to `python.defaultInterpreterPath`
File being referenced - [Gitpod settings.json](https://github.com/numpy/numpy/blob/main/tools/gitpod/settings.json).

I tried this setting on a my Fork and it seems to have set the default `vscode workspace environment` to our required one `numpy-dev`.
